expression/builtin_arithmetic_vec_test.go

```diff
@@ -37,6 +38,12 @@ var vecBuiltinArithmeticCases = map[string][]vecExprBenchCase{
 	ast.And:   {},
 	ast.Plus: {
 		{retEvalType: types.ETReal, childrenTypes: []types.EvalType{types.ETReal, types.ETReal}},
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt, types.ETInt},
```

Comment:
Please add more test cases which include unsigned types.
You can use `childrenFieldTypes` instead of `retEvalType` to indicate unsigned types.

Reply:
Done